Refresher/UI/EmulatorPatchForm.cs

```diff
@@ -1,17 +1,26 @@
+using System.Diagnostics;
 using Eto;
 using Eto.Forms;
 using Refresher.Accessors;
+using Refresher.UI.Items;
 
 namespace Refresher.UI;
 
 public class EmulatorPatchForm : IntegratedPatchForm
 {
     private FilePicker _folderField = null!;
+    private CheckBox _outputRpcs3Patch = null!;
+    private TextBox _ppuHash = null!;
+    private TextBox _gameVersion = null!;
```

Comment:
I feel like creating patch files should be a different form entirely. Maybe EBOOT patching can be under an 'advanced' button?

Reply:
i think that is a good idea yeah, i'll update the guides after this PR is merged